_episodes/39.md

```diff
@@ -0,0 +1,273 @@
+
+Data Fabric is an attempt to provide government agencies the ability to set up
+a data mesh that is backed by Trino. Trino fits well in this narrative as it
+provides SQL-over-everything. Data analysts and data scientists only need to
+know SQL.
+
+Data Fabric MVP is an end-to-end DataOps capability that can be deployed at the
+edge, in the cloud, and in disconnected environments within minutes. It provides
+a single control plane for normalizing and combining disparate data lakes, 
+platforms, silos, and formats into SQL using Trino for batch data and Apache 
+Pinot for user facing streaming analytics. 
+
+Data Fabric is driven by cloud native policy using Open Policy Agent (OPA) 
+integrated with Trino and Kafka to provide row and column level obfuscation. It
+provides enterprise data catalog to view data lineage, properties, and data
+owners from multiple data platforms. ([Source](https://datafabric.goraft.tech/))
+
+### Security concerns around Trino
+
+A common first question the Raft team gets asked is around Trino being a high
+security concern. The idea that Trino can connect to multiple data sources from
+one location brings up fear that individuals may gain access to information at
+a higher classification level than they have. The team has to educate the
+different users on the best practices and how to ensure this problem doesn't
+occur. You will need a separate deployment of Data Fabric for each 
+classification level and correctly identify policies in OPA that restrict
+visibility to information above a users' clearance.
+
+### Iron bank container repository
+
+Iron Bank is a groundbreaking central repository of digitally-signed container 
+images, including open-source and commercial off-the-shelf software, hardened to
+the DoD’s exacting specifications. Approved containers in Iron Bank have
+DoD-wide reciprocity across all classifications, accelerating down to weeks a 
+security process that can otherwise take months or even years.
+
+To be considered for inclusion into Iron Bank, container images must meet
+rigorous DoD software security standards. It is an extensive, continuous,
+complicated effort for even the most sophisticated IT teams. Continuously
+maintaining and managing hardening pipelines while incorporating evolving DoD
+specifications and addressing new vulnerabilities (CVEs) can severely stretch
+your resources, even if you have advanced tooling and experience in-house. 
+([Source](https://oteemo.com/accelerate-to-iron-bank/))
+
+The Trino Docker image 
+[is available in the Iron Bank](https://repo1.dso.mil/dsop?filter=trino) and is
+maintained by folks at Booz Allen Hamilton. Their hard work makes it possible
+for Trino to be deployed in DoD environments.
+
+## Pull requests of the episode: PR 13354: Add S3 Select pushdown for JSON files
+
+This months [PR of the episode](https://github.com/trinodb/trino/pull/13354) 
+was contributed by [preethiratnam](https://github.com/preethiratnam). Enable S3
```

Comment:
```suggestion
was contributed by [preethiratnam](https://github.com/preethiratnam): Enable S3
```
Maybe put the link to the PR on the title? It's a little weird just having it floating out in space.

Reply:
Oops, I meant to actually make that a sentence, not the title :).